packages/web/src/scss/components/Accordion/_Accordion.scss

```diff
@@ -0,0 +1,98 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../tools/typography';
+@use '../../tools/reset';
+
+.Accordion__itemHeader {
+    position: relative;
+    display: flex;
+    gap: theme.$accordion-header-gap;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 100%;
+    padding: theme.$accordion-header-padding-y theme.$accordion-header-padding-x;
+    margin-bottom: 0;
+    border-radius: theme.$accordion-border-radius;
+    background-color: theme.$accordion-item-background-color-default;
+
+    &:hover {
+        background-color: theme.$accordion-item-background-color-hover;
+    }
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        right: theme.$accordion-header-padding-x;
+        left: theme.$accordion-header-padding-x;
+        display: block;
+        height: 0;
+        border-bottom: theme.$accordion-divider-width theme.$accordion-divider-style theme.$accordion-divider-color;
+    }
+
+    &:hover::before {
+        border-bottom-color: transparent;
+    }
+}
+
+.Accordion__itemToggle {
+    @include reset.button();
+    @include typography.generate(theme.$accordion-header-typography);
+
+    flex: initial;
+    text-align: left;
+
+    &:first-of-type::before {
+        content: '';
+        position: absolute;
+        z-index: 0;
+        display: block;
+        inset: 0;
+        width: 100%;
+        height: 100%;
+        border-radius: theme.$accordion-border-radius;
+    }
+
+    &[aria-expanded='true'] {
+        @include typography.generate(theme.$accordion-header-typography-active);
+    }
+
+    // stylelint-disable selector-max-class -- We have to control the state of the Icon
+    &[aria-expanded='true'] + .Accordion__itemSide .Accordion__itemIcon {
+        transform: rotate(180deg);
```

Comment:
While my other comments target rather subtle details and the efficiency of the code, this actually can become an issue in future. We have two different icons in the design: `chevron-down` and `chevron-up`. While the transform works or this pair of icons, it will not be usable if someone decides to use `plus` and `minus` icons.

I don't insist on solving it during this PR, but I'd like to talk about it at least.

Reply:
Yes you're right. That's why I proposed from the beginning that the icon should be a pseudo element, so that the user doesn't have to put the icon there every time during implementation (for HTML) and also for this reason. But if the user is based on the documentation during the implementation, he can clearly see that the `chevron-down` => icon is used for the closed (default) state.

But it could be mentioned in the documentation that we "recommend" using `chevron-down' in the implementation.